extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -714,6 +714,9 @@ export class AmpConsent extends AMP.BaseElement {
    * @return {?JsonObject|undefined}
    */
   configureMetadataByConsentString_(opt_metadata, opt_consentString) {
+    if (!opt_metadata) {
```

Comment:
IMO the can be combined with `return opt_metadata` unless you want to handle `opt_metadata = null` case specially. 

What about 
```
if (opt_metadata && !isObject(opt_metadata ..) {
  //user error;
  return undefined
}
return opt_metadata;
```

Reply:
ConsentStringType PR will change this method, so I'm doing a little thinking ahead even if it's not the cleanest.